src/drawing/DrawingPath.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { Position } from '../@types';
+import './DrawingPath.scss';
+
+export type Props = {
+    borderStrokeWidth?: number;
+    decorated?: boolean;
+    points: Position[];
+};
+
+// We use cubic Bezier curves to generate path commands for better smoothing.
+// For every data point, we generate extra two points using adjacent data points.
+// Then we use the first generated point and the data point as two control points
+// and the second generated point as the destination point to form a (C)urve command.
+export const getPathCommands = (points: Position[]): string => {
+    if (!points || !points.length) {
+        return '';
+    }
+
+    const { x: startX, y: startY } = points[0];
+    const d = points
+        .map(({ x, y }, index, array) => {
+            const prevPoint = array[index - 1];
+            const nextPoint = array[index + 1];
+            if (!prevPoint || !nextPoint) {
+                return '';
+            }
+
+            const xc1 = (x + prevPoint.x) / 2;
+            const yc1 = (y + prevPoint.y) / 2;
+            const xc2 = (x + nextPoint.x) / 2;
+            const yc2 = (y + nextPoint.y) / 2;
+
+            return `C ${xc1} ${yc1}, ${x} ${y}, ${xc2} ${yc2}`;
+        })
+        .join(' ');
+
+    return `M ${startX} ${startY} ${d}`;
+};
+
+export const DrawingPath = ({ borderStrokeWidth = 0, decorated = false, points }: Props): JSX.Element => {
+    const pathCommands = getPathCommands(points);
```

Comment:
Does this need to be a separate function or can it happen inline? Ideally, we should test components as a black box with Enzyme versus testing the inner logic.

Reply:
I'll need it exposed for create drawing, so leaving it as an export is fine, or I can export when when I integrate create drawing code

Reply:
It seems like we should try to reuse the component itself for drawing creation, similar to how we reuse RegionRect during region creation.

Reply:
Discussed offline. We will likely want to set the `d` attribute of the path directly via ref, since that will allow us to avoid React's render loop and improve performance. We'll need to keep `getPathCommands` separate to support that.